charts/flux2/values.yaml

```diff
@@ -13,6 +13,9 @@ installCRDs: true
 # eg: http://notification-controller.[NAMESPACE].svc.[CLUSTERDOMAIN]
 eventsaddr: http://notification-controller/
 
+checker:
```

Comment:
```suggestion
cli:
```

Reply:
Thanks for the comment, they are resolved.